From ccf3f6f21a59b619fa25c4d0f00d5f83693a8518 Mon Sep 17 00:00:00 2001
From: AXIM0S <aminurfisa3@gmail.com>
Date: Sat, 8 Aug 2020 18:05:38 -0400
Subject: [PATCH] Add Gearlock to device.mk

---
 device.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/device.mk b/device.mk
index f16cba2..868b417 100644
--- a/device.mk
+++ b/device.mk
@@ -232,6 +232,10 @@ PRODUCT_COPY_FILES += \
     $(foreach f,$(wildcard $(LOCAL_PATH)/alsa/*),$(f):$(subst $(LOCAL_PATH),system/etc,$(f))) \
     $(foreach f,$(wildcard $(LOCAL_PATH)/idc/*.idc $(LOCAL_PATH)/keylayout/*.kl),$(f):$(subst $(LOCAL_PATH),system/usr,$(f))) \
     x86_installer/bin/Androidx86-Installv26.0003.exe:Androidx86-Installv26.0003.exe 
+    
+# Gearlock
+PRODUCT_COPY_FILES += \
+    vendor/gearlock/out/gri/gearlock:gearlock
 
 # WIDEVINE OVERRIDES
 # PRODUCT_COPY_FILES += \
-- 
2.17.1

